examples/fluids/navierstokes.h

```diff
@@ -275,21 +273,24 @@ struct Physics_private {
   CeedContextFieldLabel ics_time_label;
 };
 
+PetscErrorCode BoundaryConditionSetUp(User user, ProblemData problem, AppCtx app_ctx, SimpleBC bc);
+
 typedef struct {
   CeedQFunctionUser    qfunction;
   const char          *qfunction_loc;
   CeedQFunctionContext qfunction_context;
 } ProblemQFunctionSpec;
 
 // Problem specific data
-typedef struct ProblemData_private *ProblemData;
 struct ProblemData_private {
   CeedInt              dim, q_data_size_vol, q_data_size_sur, jac_data_size_sur;
   CeedScalar           dm_scale;
   ProblemQFunctionSpec ics, apply_vol_rhs, apply_vol_ifunction, apply_vol_ijacobian, apply_inflow, apply_outflow, apply_freestream, apply_slip,
       apply_inflow_jacobian, apply_outflow_jacobian, apply_freestream_jacobian, apply_slip_jacobian;
-  bool      non_zero_time;
-  PetscBool bc_from_ics, use_strong_bc_ceed, uses_newtonian;
+  bool          non_zero_time;
+  PetscBool     bc_from_ics, use_strong_bc_ceed, uses_newtonian;
```

Comment:
I've been trying to push us to start all bools with verbs since most of them already do